apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant/Defendant.tsx

```diff
@@ -215,6 +222,17 @@ const Defendant: React.FC = () => {
     }
   }
 
+  const options = useMemo(
```

Comment:
Instead of maintaining two lists, we map through the cases an get the label from `caseTypes`